processor/src/main/java/org/mapstruct/ap/internal/model/ContainerMappingMethod.java

```diff
@@ -36,11 +36,8 @@
  *
  * @author Filip Hrisafov
  */
-public abstract class ContainerMappingMethod extends MappingMethod {
```

Comment:
Ok.. but if the getDefault() moves what does remain here? And does it justify to have a common base class. I'm not in favor of long inheritance chains, and prefer composition of inheritance if possible. Anyway. Don't consider this as blocker.